src/Api/quizzes.ts

```diff
@@ -1,13 +1,19 @@
 import axios from "axios";
-import { useQuery } from "@tanstack/react-query";
+import { useMutation, useQuery } from "@tanstack/react-query";
 import Cookies from "js-cookie";
+import { QuizType } from "../types";
+
+const reqOptions = {
```

Comment:
note for later: This will likely be used in other places as well. It would better to define and export it from a location that makes sense.